format-json/src/test/java/io/cdap/plugin/format/json/JsonInputFormatProviderTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2018-2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+
+package io.cdap.plugin.format.json;
+
+import io.cdap.cdap.api.data.schema.Schema;
+import io.cdap.cdap.etl.api.validation.FormatContext;
+import io.cdap.cdap.etl.api.validation.ValidationFailure;
+import io.cdap.cdap.etl.mock.validation.MockFailureCollector;
+import io.cdap.plugin.format.input.PathTrackingConfig;
+import io.cdap.plugin.format.json.input.JsonInputFormatProvider;
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class JsonInputFormatProviderTest {
+    @ClassRule
+    public static final TemporaryFolder TMP_FOLDER = new TemporaryFolder();
+
+    @Test(expected = Test.None.class)
```

Comment:
don't use expected when there is no expected exception

Reply:
Removed expected